gpflow/models/cglb.py

```diff
@@ -159,9 +169,10 @@ def quad_term(self, common: SGPR.CommonTensors) -> tf.Tensor:
 
         return -ub
 
+    @inherit_check_shapes
     def predict_f(
         self,
-        xnew: InputData,
+        Xnew: InputData,
```

Comment:
Hi, may I ask about this? This would be backwards incompatible change for users explicitly specifying `xnew` as a keyword argument. What is the motivation for this change? Are we sure nothing would break on the user side if we rename this?

Thank you!

Reply:
> Hi, may I ask about this? This would be backwards incompatible change for users explicitly specifying `xnew` as a keyword argument. What is the motivation for this change? Are we sure nothing would break on the user side if we rename this?
> 
> Thank you!

Yes, I'm aware this is a breaking change, and no I'm not sure I'm not breaking anything. BUT the current state is that it's incompatible with all the other models where this is called `Xnew` which prevents polymorphism so... today you cannot write:
```python
    model.predict_f(xnew=my_new_x)
```
without knowing the type of `model`, because it's called `Xnew` in most of the models, but `xnew` in the cglb...

GPflow is full of this kind of stuff, and while I try to keep breaking changes reasonably rare, we'll probably need to be able to clean this kind of stuff up if we want to be able to continue development while remaining reasonably sane.

Reply:
Thank you @jesnie , I understand.

Would you like to take this in two steps, and only add the shape checks in this PR whilst renaming the input variable in other one? I can take care of the second PR, given that you have already done all the work.

Thank you, and please let me know what you think.

Reply:
> Thank you @jesnie , I understand.
> 
> Would you like to take this in two steps, and only add the shape checks in this PR whilst renaming the input variable in other one? I can take care of the second PR, given that you have already done all the work.
> 
> Thank you, and please let me know what you think.

I'm a big fan of "one thing per PR", but the `@inherit_check_shapes` won't work without the renaming, because it's looking for an argument called `Xnew`, which should have been inherited from the parent. There are ways to deal with that, but it seems like overkill to me, for the sake of the naming of one argument...

Reply:
Makes sense, thank you!

I guess it would be useful to add a line to the release notes for that change.